modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/delegate/delegate.es.js

```diff
@@ -54,3 +54,4 @@ function delegate(element, eventName, selector, callback) {
 }
 
 export default delegate;
+export {delegate};
```

Comment:
> The reason why I added the export {delegate} alongside the export default delegate is because that's the approach taken in other utilities, for example in debounce.es.js

See https://github.com/liferay-frontend/liferay-portal/pull/439#discussion_r511969174

> Do not export the same thing twice; I know you're just following the old convention that was used in some Liferay codebases, but we now consider it an anti-pattern. [The linter will prevent this in the future](https://github.com/liferay/liferay-frontend-projects/issues/4).

Reply:
Beware of cargo-culting: consistency is great and all, but it's worth taking the time to understand _why_ something is being done in a certain way. In this case, we concluded there's no good reason to do this any more (hence the lint, which I guess I have to prioritize because this keeps coming up).

Reply:
Makes sense, thanks guys!